bcipy/acquisition/protocols/lsl/lsl_client.py

```diff
@@ -26,24 +24,23 @@ class LslAcquisitionClient:
 
     Parameters
     ----------
-        max_buflen: the maximum length, in seconds, of data to be queried.
```

Comment:
could this be buffer_len?

Reply:
I was using the parameter name used by `StreamInlet`, but I agree that it doesn't follow our naming conventions. How about `max_buffer_len`? I think keeping 'max' in there is important since it could be smaller than this at any given time.